Prace_domowe/SVM.ipynb

```diff
@@ -0,0 +1,1998 @@
+{
```

Comment:
<p>This is how we do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>